core/templates/components/code-mirror/codemirror.component.ts

```diff
@@ -20,31 +20,46 @@ import { AfterViewInit, Component, EventEmitter, Input, OnChanges, Output, Simpl
 import { downgradeComponent } from '@angular/upgrade/static';
 import { CodemirrorComponent } from '@ctrl/ngx-codemirror';
 
+export interface CodeMirrorMergeViewOptions {
```

Comment:
I think we should avoid exporting the interface if it is not being used outside this file

Reply:
done